From b0e30888de3d728b6f1ecebdefc2a60c8a57a27b Mon Sep 17 00:00:00 2001
From: TheAtt1la <alittauwu@gmail.com>
Date: Sat, 17 Dec 2022 02:12:23 -0300
Subject: [PATCH 2/3] remove duplicate ro.adb.secure

---
 core/main.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index 3631758..eee60a8 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -369,12 +369,12 @@ enable_target_debugging := true
 tags_to_install :=
 ifneq (,$(user_variant))
   # Target is secure in user builds.
-  ADDITIONAL_SYSTEM_PROPERTIES += ro.secure=1
+#  ADDITIONAL_SYSTEM_PROPERTIES += ro.secure=1
   ADDITIONAL_SYSTEM_PROPERTIES += security.perf_harden=1
 
-  ifeq ($(user_variant),user)
-    ADDITIONAL_SYSTEM_PROPERTIES += ro.adb.secure=1
-  endif
+ # ifeq ($(user_variant),user)
+ #   ADDITIONAL_SYSTEM_PROPERTIES += ro.adb.secure=1
+ # endif
 
   ifeq ($(user_variant),userdebug)
     # Pick up some extra useful tools
-- 
2.38.1


